packages/donor/src/screens/questionnaire/QuestionnaireScreen.tsx

```diff
@@ -155,28 +76,17 @@ export default function QuestionnaireScreen({
       options={YesNoOptions}
       value={isRightAge}
       onChange={setIsRightAge}
-      label={"האם אתה בטווח הגילאים 17-65?"}
+      label={"האם הנך מעל גיל 17?"}
     />
   );
 
-  const [isPregnantEver, setIsPregnantEver] = useState("");
-  const IsPregnantEver = (
+  const [wasPregnant, setWasPregnantEver] = useState("");
```

Comment:
Why is this a string value?

Reply:
I am not sure, but I didn't change this.
We can change this in a different PR